atlasdb-commons/src/main/java/com/palantir/util/debug/StackTraceUtils.java

```diff
@@ -162,12 +181,16 @@ public static String threadToString(ThreadInfo info, Object[] lockedMonitors, in
         StackTraceElement[] stackTrace = info.getStackTrace();
         for (int i = 0; i < stackTrace.length && i < MAX_FRAMES; i++) {
             StackTraceElement ste = stackTrace[i];
-            sb.append(INDENT + "at " + ste.toString());
+            sb.append(INDENT + "at ").append(ste.toString());
```

Comment:
Yes, this is mostly just micro-optimization cleanup to avoid unnecessary allocations since we're already building a large string.

Any of the constant expressions will be automatically concatenated by javac during compilation. If there are concatenations of non-constant expressions, those are evaluated at runtime and will allocate intermediate strings passed to append. See https://docs.oracle.com/javase/specs/jls/se11/html/jls-15.html#jls-15.18.1